R/ttestindependentsamples.R

```diff
@@ -524,8 +525,9 @@ ttestIndependentMainTableRow <- function(variable, dataset, test, testStat, effS
         mean <- mean(groupDataOm)
         std  <- sd(groupDataOm)
         sem  <- std / sqrt(n)
+        cv   <- std/abs(mean)
```

Comment:
Also, for consistency with jaspDescriptives, do not use the absolute value of the mean (https://github.com/jasp-stats/jaspDescriptives/blob/1aea742538ea770ba2ee52caaf9fd7cc6096ef1a/R/descriptives.R#L439)

Reply:
Oh sorry, I missed this comment. I noticed that in jaspDescriptives indeed the mean is used instead of the absolute mean, but I thought the absolute value would be more fit for (relative) dispersion.  But I will change this first thing in the morning!

Reply:
yeah no worries, I did provide this comment after the first review, so it's easy to miss. Thanks for changing it.